### PR TITLE
Registeration

### PR DESCRIPTION
**Pull request by OkechukwuNnujiJohn**
## Description

In this pull request, I am working on making the user-registeration and login work but i seem to run into an error with fetching the local host
The user will be navigated to the home page

## Milestones
User-registeration, allowing the user signup and create an account and login with the account. 

## Resources
links to tutorials, code snippets, inspirations: (https://github.com/codepath/auth-express-react-guide)

## Test Plan
Screenshot 2023-07-06 at 11 02 50 PM
Screenshot 2023-07-06 at 11 02 25 PM
Screenshot 2023-07-06 at 11 02 03 PM
Screenshot 2023-07-06 at 11 01 37 PM

### CODE REVIEW

**Comment by JinyueHu** [comment]
Hi John, could you split into smaller pull request? Or do you want to merge it first and then keep pull request smaller?

**Comment by OkechukwuNnujiJohn** [comment]
Hi,

Let’s merge it first and then keep then following I will make the pull
request smaller.

File: J.Thrift/J.Thrift-api/app.js [hunk 1]
```diff
@@ -21,56 +54,115 @@ app.get('/users', async (req, res) => {
...
-      });
-      res.json(posts);
+      const carts = await Cart.findAll();
+      res.json(carts);
     } catch (err) {
       res.status(500).json({ message: err.message });
     }
   });
 
+  // //Route to get all Cart
```
Comment (JinyueHu) [inline J.Thrift/J.Thrift-api/app.js:77]: please delete those commented out code next time in PR :D

File: J.Thrift/J.Thrift-api/node_modules/@mapbox/node-pre-gyp/lib/util/napi.js
Comment (JinyueHu) [inline J.Thrift/J.Thrift-api/node_modules/@mapbox/node-pre-gyp/lib/util/napi.js:1]: Please try to make the PR as small as possible, that might be easier for review. 

Also separate PR based on different purpose and theme.

**Review by JinyueHu — APPROVED** [review]
Approve to merge since we had a live demo today, please take the comments and feedbacks in your following PRs! Thank you!

**Comment by OkechukwuNnujiJohn** [comment]
Understood. Thank you

**Pull request merged**